lib/senkyoshi/models/wikipage.rb

```diff
@@ -2,6 +2,8 @@
 
 module Senkyoshi
   class WikiPage < Content
+    include Senkyoshi
+
```

Comment:
Not sure if you need this include Senkyoshi in this file. None of the others models have it.